protocol/chat.go

```diff
@@ -636,3 +636,12 @@ func stringSliceContains(slice []string, item string) bool {
 	}
 	return false
 }
+
+func ConvertChatType(chatType ChatType) chatContext {
```

Comment:
`ConvertChatType` might not be the best name. It conveys that we are changing the type from what it was before to something else, but this is more a mapping.

I suggest instead something like `GetChatContextFromChatType`